kafka-commons/src/main/java/uk/gov/homeoffice/digital/sas/kafka/consumer/configuration/KafkaConsumerConfig.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.homeoffice.digital.sas.kafka.consumer.configuration;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.MeterRegistry;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan("uk.gov.homeoffice.digital.sas.kafka.consumer")
+public class KafkaConsumerConfig {
+
+  @Value("${actuator.kafka.failure.url}") String actuatorKafkaFailureUrl;
+  @Value("${actuator.error.type}") String actuatorErrorType;
+  @Value("${actuator.failure.description}") String actuatorKafkaFailureDescription;
```

Comment:
Now that meterRegistry is injected through the constructor, it would be consistent to also inject these properties in the same fashion?

Reply:
Have injected these properties into the constructor